Xamarin.Forms.Core/SwipeItem.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.ComponentModel;
+using System.Windows.Input;
+
+namespace Xamarin.Forms
+{
+	public interface ISwipeItem
```

Comment:
one public type per file is usually better

Reply:
Separated in different files.